backend/core/serializers.py

```
@@ -959,6 +959,12 @@ class EvidenceWriteSerializer(BaseModelSerializer):
     requirement_assessments = serializers.PrimaryKeyRelatedField(
         many=True, queryset=RequirementAssessment.objects.all(), required=False
     )
+    findings = serializers.PrimaryKeyRelatedField(
+        many=True, required=False, queryset=Finding.objects.all()
```

Ruff (0.11.9)

`Finding` may be undefined, or defined from star imports

(F405)